### DISCUSSION TITLE
Thermal conductivity：Dpmd+EMD(Green-kubo)

### DISCUSSION THREAD

**Discussion by Suyoung999999999** [deepmodeling/dpgen#1967] [Q&A]
### Summary

According to some studies, the method based on Green-kubo combined with Dpmd is a feasible scheme to calculate the thermal conductivity of some structures. However, the current description of training data collection using deepmd-kit and dpgen for Dp potential fitting suitable for thermal conductivity calculation is not very clear in most literature. So I have two questions, in the form of examples. Taking the calculation of the thermal conductivity of single crystal Si at 300K as an example, 1. If deepmd-kit is used to fit the Dp potential, what DFT data should be collected by vasp as training data and test data, so that a potential function accurately describing its thermal conductivity can be fitted well? 2. Using the calculation method of dpgen, how to set the initial training data and MD iteration part? Is the ensemble of MD part of dpgen npt or nvt? How to set up each iteration reasonably? Thank you!

### DP-GEN Version

2.2.7

### Details

Can an example be given in bohrium platform for the calculation of thermal conductivity based on Green-kubo method (Dp potential + vasp +lammps) from data collection to thermal conductivity calculation? Or give a feasible Dp training scheme? thank you.

**Comment by Wanwan-Laang** [comment]
you can directly use the DeePMD-kit to training the potential as long as you have enough data. Otherwise, the DP-GEN can help you obtain more configuration. In model divation part, you can decide which npt or nvt, which depend on your study. BTW, there are many good literatures study on the thermal conductivity calculation with DPMD, most of them express very clear. or you can also search some the notebook from bohrium platform.

[a comment by an automated account is not shown]
